### Polygon64 message
#
# A specification of a polygon with float64 points and where the first and last
# points are assumed to be connected if not equal.
#
geometry_msgs/Point[] points  # [meters, meters, meters]
